.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = crlf
```

Comment:
The repository most certainly does not use crlf line endings. It may be that git adds those when you check out on windows, but this should not be in the editor config.